src/System Application/App/User Settings/src/UserSettings.Page.al

```diff
@@ -114,6 +114,14 @@ page 9204 "User Settings"
                         Caption = 'Teaching Tips';
                         ToolTip = 'Specifies whether to display short messages that inform, remind, or teach you about important fields and actions when you open a page.';
                     }
+                    field("Legacy Action Bar"; Rec."Legacy Action Bar")
+                    {
+                        ApplicationArea = All;
+                        Caption = 'Legacy Action Bar';
+#pragma warning disable AA0219
+                        ToolTip = 'Revert the action bar to the legacy terms and layout. For example, the action bar won''t be pinned on pages, the Home tab is named Process, and split buttons become multiple actions, which means more clicks. Also, your options for personalization will be limited.';
```

Comment:
Specifies whether to revert...
This way you remove the warning disablement, especially when it's not needed.

Reply:
This was given by Jacob, sticking with it.